### DISCUSSION TITLE
Remove Threading in Sink in favor of Serilog.Sinks.Async?

### DISCUSSION THREAD

**Discussion by Nefarion** [augustoproiete/serilog-sinks-richtextbox#65] [Ideas]
[Serilog.Sinks.File](https://github.com/serilog/serilog-sinks-file#performance) for example uses [Serilog.Sinks.Async](https://github.com/serilog/serilog-sinks-async#getting-started) to give the user the choice to either operate on the main thread or in a background thread.

**Comment by augustoproiete** [comment]
Hey @Nefarion, thanks for the suggestion. What benefits would that give users of this sink? Are you looking for any behavior in particular in terms of UX?
